ci-operator/config/jaegertracing/jaeger-operator/jaegertracing-jaeger-operator-main__jaeger-product-ocp-4.14-lp-interop.yaml

```diff
@@ -37,18 +37,19 @@ tests:
-          {"step": "distributed-tracing-install-*", "failure_type": "test_failure", "classification": "Distributed tracing operators", "jira_project": "TRACING", "jira_epic":"LPTOCPCI-290"},
-          {"step": "operatorhub-subscribe-*", "failure_type": "pod_failure", "classification": "Distributed tracing dependency operators", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
-          {"step": "operatorhub-subscribe-*", "failure_type": "test_failure", "classification": "Distributed tracing dependency operators", "jira_project": "TRACING", "jira_epic":"LPTOCPCI-290"},
-          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
-          {"step": " distributed-tracing-tests-*", "failure_type": "pod_failure", "classification": "Jaeger e2e tests", "jira_project": "LPTOCPCI", "jira_epic": "LPTOCPCI-290"},
-          {"step": " distributed-tracing-tests-*", "failure_type": "test_failure", "classification": "Jaeger e2e tests", "jira_project": "TRACING", "jira_epic":"LPTOCPCI-290"}
+          {"step": "build-logs", "failure_type": "pod_failure", "classification": "Jaeger operator images", "jira_project": "LPTOCPCI"},
+          {"step": "build-logs", "failure_type": "test_failure", "classification": "Jaeger operator images", "jira_project": "TRACING"},
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure provisioning", "jira_project": "LPTOCPCI"},
+          {"step": "install", "failure_type": "pod_failure", "classification": "Jaeger operator install", "jira_project": "LPTOCPCI"},
+          {"step": "install", "failure_type": "test_failure", "classification": "Jaeger operator install", "jira_project": "TRACING"},
+          {"step": "distributed-tracing-install-*", "failure_type": "pod_failure", "classification": "Distributed tracing operators", "jira_project": "LPTOCPCI"},
+          {"step": "distributed-tracing-install-*", "failure_type": "test_failure", "classification": "Distributed tracing operators", "jira_project": "TRACING"},
+          {"step": "operatorhub-subscribe-*", "failure_type": "pod_failure", "classification": "Distributed tracing dependency operators", "jira_project": "LPTOCPCI"},
+          {"step": "operatorhub-subscribe-*", "failure_type": "test_failure", "classification": "Distributed tracing dependency operators", "jira_project": "TRACING"},
+          {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPTOCPCI"},
+          {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
+          {"step": " distributed-tracing-tests-*", "failure_type": "pod_failure", "classification": "Jaeger e2e tests", "jira_project": "LPTOCPCI"},
+          {"step": " distributed-tracing-tests-*", "failure_type": "test_failure", "classification": "Jaeger e2e tests", "jira_project": "TRACING"}
```

Comment:
extra spaces in beginning `" distributed-tracing-tests-*",`

Reply:
Fixed